modules/frontend/journal/JournalFrontendModule.php

```diff
@@ -120,6 +130,11 @@ public function renderBackend() {
 		foreach(JournalQuery::create()->orderByName()->find() as $oJournal) {
 			$aJournalOptions[$oJournal->getId()] = TranslationPeer::getString('wns.journal.journal_name', null, null, array('name' => $oJournal->getName()));
 		}
+		$aLimitOptions = array();
+		foreach(range(1,3) as $iLimit) {
+			$aLimitOptions[$iLimit] = $iLimit;
+		}
```

Comment:
```suggestion
		$aLimitOptions = array_combine(range(1,3), range(1,3));
```

Reply:
Why limit the limit to max three items?

Reply:
Simply because I'm never quite sure how to do it: a drop down is easier than a number that requires validation again. And also it's not intended to show lots of entries, then we might as well show the blog overview list, no? What do you suggest? (1,2,3,5,10...)